modules/core/src/main/java/org/apache/synapse/core/axis2/ProxyService.java

```diff
@@ -239,6 +235,8 @@
 
     private boolean wsdlPublished;
 
+    private String carName = "";
```

Comment:
Shall we rename this variable name ? Let's use a generic name like ArtifactContainer, ParentArtifact (without using the name CAR) This need to be done everywhere